.circleci/config.yml

```diff
@@ -473,5 +486,7 @@ jobs:
           variant: "Release"
       - update-version-name
       - track-metrics
+      - init-aws
+      - check-api
```

Comment:
What is the reason/requirement for adding this to:  `build-sdk-release`?
Depending on how long it runs, I would opt to either add it to verify-code
or as a standalone `verify-api` (similar to verify-docs).

![image](https://user-images.githubusercontent.com/2151639/132181625-f8dc56a6-0805-4903-9793-bea92c915dcb.png)

Reply:
I would prefer a standalone `verify-api`. Will add it later.